include/endian.h

```diff
@@ -29,9 +29,20 @@
 
 #include <machine/endian.h>
 
-#define __LITTLE_ENDIAN 1234
-#define __BIG_ENDIAN 4321
-#define __PDP_ENDIAN 3412
+#include <uk/config.h>
+
+#if ((defined CONFIG_ARCH_X86_64) || (defined CONFIG_ARCH_ARM_64))
```

Comment:
Same as the lwip PR, I think we can skip this ifdef. It's expected to deal with endianess-related headers when porting a new architecture, it;s ok not to error.